python/ml4ir/base/model/relevance_model.py

```diff
@@ -234,16 +246,25 @@ def fit(
             monitor_mode=monitor_mode,
             monitor_metric=monitor_metric,
             patience=patience,
-            track_experiment=track_experiment,
         )
         if self.is_compiled:
-            self.model.fit(
+            history = self.model.fit(
                 x=dataset.train,
                 validation_data=dataset.validation,
                 epochs=num_epochs,
                 verbose=True,
                 callbacks=callbacks_list,
             )
+
+            # Write metrics for experiment tracking
+            train_metrics = dict()
+            for metric, value in history.history.items():
+                if not metric.startswith("val_"):
+                    train_metrics["train_{}".format(metric)] = value[0]
+                else:
+                    train_metrics[metric] = value[0]
+
+            return train_metrics
```

Comment:
Returning metrics to be stored for experiment tracking. Since we are not using weights and biases, I am creating a dict manually.

Reply:
This info needs to be part of python doc. Ie explicitly define what we are returning.

Reply:
Added.